src/containers/StartupProjects/StartupProject.js

```diff
@@ -27,21 +27,26 @@ export default function StartupProject() {
           >
             {bigProjects.subtitle}
           </p>
-          <div className="startup-projects-main">
-            <div className="startup-project-text">
+
+          <div className="project-cards-div">
```

Comment:
I think we should use a classname like `projects-container`, shorter and informative.